### ISSUE TITLE
False-positive: empty_line_in_statement

### ISSUE THREAD

**Issue by lucasborin** [abaplint/abaplint#2586]
```ABAP
  METHODS:
    "! <p class="shorttext synchronized" lang="en">EML Modify</p>
    "! @parameter entities | <p class="shorttext synchronized" lang="en">Entities to modify</p>
    "! @parameter result | <p class="shorttext synchronized" lang="en">EML Responses</p>
    modify
      IMPORTING
        entities      TYPE if_cpe_formula_bo_types=>update-entities-roottermexchrateperiods
      RETURNING
        VALUE(result) TYPE if_cpe_formula_bo_types=>response-early,

    "! <p class="shorttext synchronized" lang="en">EML Create</p>
    "! @parameter entities | <p class="shorttext synchronized" lang="en">Entities to create</p>
    "! @parameter result | <p class="shorttext synchronized" lang="en">EML Responses</p>
    create
      IMPORTING
        entities     TYPE if_cpe_formula_bo_types=>create-entities-roottermexchrateperiods
      RETURNING
        VALUE(result) TYPE if_cpe_formula_bo_types=>response-early,
```

![image](https://user-images.githubusercontent.com/5233413/177782693-cd822258-e0a3-42ec-bba2-d803511c002b.png)

**Comment by lucasborin** [comment]
It looks like it is reporting the empty line between the methods, but the stack trace is not accurate.

**Comment by lucasborin** [comment]
You are right. We misunderstood it. Thanks again, Lars.

**Comment by larshp** [comment]
feel free to suggest changes in the documentation, https://github.com/abaplint/abaplint/blob/e82f4faec362eb81281e9485f2fd88e4c1d68acd/packages/core/src/rules/empty_line_in_statement.ts#L25
